index.js

```diff
@@ -19,6 +19,7 @@ async function assumeRole(params) {
   const isDefined = i => !!i;
 
   const {
+    audience,
```

Comment:
This line is causing our testing to fail since it's not being used - the webIdentityToken should have already covered the audience, right? Do you remember why you added this line here?

Reply:
Thank you @enriquesantosblanco-sanuk - I worked with @alblue. It was devastating news. 
Although I cannot help with the specific code, I am happy to assist in ensuing this gets completed if needs be.

Reply:
This is devastating news... I'm so sorry, my condolences 😞

Reply:
Thank you on behalf of his entire Cloud Platform Evolution team. Knowing that his genius lives on through code is comforting to all that knew him. All the best to you and yours!